completions/carton

```diff
@@ -75,6 +75,7 @@ _comp_cmd_carton()
         exec)
             # skip all the options --, -v, etc. and identify the command name
             # position.
+            local i
```

Comment:
This seems unnecessary, `i` is localized on line 21 above, and I think the intent is to continue iterating here where the earlier loop left off.

Reply:
Ah, thank you for pointing out. I missed it. I removed the first loop in an (unsubmitted) commit [`b92f4ce`](https://github.com/scop/bash-completion/commit/b92f4ce1838a47df1c421ba1f2286f09e229caa2#diff-91682bca968cc4516eec6a97d940a3cb4af23ede5aa518e28813e6635bfbfec9), but I seemed to have missed that `i` of the first loop is reused here. I'll drop the change. I'll also update the unsubmitted commit.

Reply:
I've dropped the change from the leakvar commit.